plugins/spam.py

```diff
@@ -3,32 +3,28 @@
 from errbot import BotPlugin
 from errbot.templating import tenv
 
-from plugins import constants
+from utils.mixin import DefaultConfigMixin
 
 
-class SpammingAlert(BotPlugin):
+class SpammingAlert(BotPlugin, DefaultConfigMixin):
     """
     A plugin which alerts the user that they might be spamming.
     """
 
-    CONFIG_TEMPLATE = {
-        'MAX_MSG_LEN': constants.MAX_MSG_LEN,
-        'MAX_LINES': constants.MAX_LINES
-    }
-
     def get_configuration_template(self):
```

Comment:
Reorder class definition as `SpammingAlert(DefaultConfigMixin, BotPlugin)` and you can eliminate the `get_configuration_template` overriding completely.